CHANGELOG.md

```diff
@@ -34,16 +34,25 @@ Thank you to all who have contributed!
 ### Deprecated
 
 ### Fixed
-- Return type of `partiql-ast`'s `SqlDialect` for `defaultReturn` to be a `SqlBlock` rather than `Nothing`
-- Flatten `CASE WHEN` branch type in `PlanTyper`
 
 ### Removed
 
 ### Security
 
+### Contributors
+Thank you to all who have contributed!
+- @<your-username>
+
+## [0.14.3] - 2024-02-14
```

Comment:
Could you add a link for v0.14.3 below? https://github.com/partiql/partiql-lang-kotlin/blob/bec3a19303d0591d50f750b77c82e6028886738a/CHANGELOG.md?plain=1#L1003